doc/en/example/parametrize.rst

```diff
@@ -396,6 +396,31 @@ The result of this test will be successful:
 
     ========================== no tests ran in 0.12s ===========================
 
+Note, that each argument in `parametrize` list should be explicitly declared in
```

Comment:
I think we don't need to add this section in the docs: it is explaining a rare error, which rarely happens. If a user makes this mistake, the error message should be enough to point they in the right direction.

Reply:
OK, but maybe a few lines won't be redundant. I agree that the full section probably is an overkill

Reply:
OK, sounds good.

Reply:
Updated that in last commit!